src/_P036_FrameOLED.ino

```diff
@@ -297,6 +375,11 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
 
     case PLUGIN_WEBFORM_LOAD:
       {
+#ifdef PLUGIN_036_DEBUG
+          addLog(LOG_LEVEL_INFO, F("P036_PLUGIN_WEBFORM_LOAD ..."));
+#endif  // PLUGIN_036_DEBUG
+        P036_data_temp = new P036_data_struct();  // just temporary for Plugin_036_loadDisplayLines()
```

Comment:
There is a number of times you do a `new` on this pointer, but I do miss a few calls to delete.

Are you sure this is not causing a memory leak, every time you look at the web page for this plugin?

Reply:
In the new version there is a Delete.
I thought it will be automatically deleted while leaving the function.

Reply:
If you want that kind of behavior you have to use one of those smart pointer wrappers to store the pointer.
That's always a more safe way to store data and we could use more of them in a lot of places in the code.
But as long as you don't use one of those, you have to delete them manually.